lib/node_modules/@stdlib/blas/base/dnrm2/src/dnrm2.f

```diff
@@ -52,6 +52,20 @@
 !<
 double precision function dnrm2( N, dx, stride )
   implicit none
+  ! ..
+  ! Define a kind parameter for single precision:
+  integer, parameter :: wp = kind( 1.0d0 )
+  ! ..
+  ! Define constants:
+  real( wp ), parameter :: zero = 0.0_wp
```

Comment:
Is `real` the right function here? Based on https://gcc.gnu.org/onlinedocs/gfortran/REAL.html, I am not sure.

Reply:
What does LAPACK use here?

Reply:
Based on the `lapack` implementation this is correct: https://www.netlib.org/lapack/explore-html/d1/d2a/group__nrm2_gab5393665c8f0e7d5de9bd1dd2ff0d9d0.html#gab5393665c8f0e7d5de9bd1dd2ff0d9d0

Reply:
Also, we have used a similar approach for `dznrm2` as well: https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/blas/base/dznrm2/src/dznrm2.f#L58C1-L59C1

Reply:
Thanks for confirming.